protocol/src/main/java/org/geysermc/mcprotocollib/protocol/packet/ingame/serverbound/ServerboundChatPacket.java

```diff
@@ -26,12 +26,11 @@ public ServerboundChatPacket(ByteBuf in, MinecraftCodecHelper helper) {
         this.message = helper.readString(in);
         this.timeStamp = in.readLong();
         this.salt = in.readLong();
-        if (in.readBoolean()) {
-            this.signature = new byte[256];
-            in.readBytes(this.signature);
-        } else {
-            this.signature = null;
-        }
+        this.signature = helper.readNullable(in, inBuf -> {
```

Comment:
inBuf -> buf to be consistent.

Reply:
I really having to name them. It's all literally writing to the same buf.